src/css/_custom.css

```diff
@@ -1,27 +1,52 @@
 .lh-f1 {
-  line-height: 56px;
+	line-height: 56px;
```

Comment:
@sophielevens @maxgerber Can we all use prettier with 2 space indentation? This will make code reviewing easier!